src/olympia/reviewers/views.py

```diff
@@ -1261,6 +1261,9 @@ def decision_review(request, decision_id):
             case 'no':
                 new_decision = ContentDecision.objects.create(
```

Comment:
I feel we should we have a `deny(action_to_use_instead)` method or something along those lines (with a better name) on `ContentDecision` to move all the code dealing with somebody "cancelling" a held decision.

Doesn't have to be in this PR though.

Reply:
AMO_APPROVE is the exceptional case, where there's only one action and policy - to generalize it we'd need to build more of a review tool that would allow a policy selection and action.  (That would also be a replacement for the "forward" action, which is a bit of hack)

Reply:
So, a `deny_held_action()` on `ContentDecision` is possible, which would be the same code in the view, but moved. A more general `override_action(action, policies)` is dependent on some more form/view work.  I'll log an issue.

Reply:
https://github.com/mozilla/addons/issues/15396